discovery/spec.md

```diff
@@ -132,6 +132,10 @@ Service:
   "url": "[unique URL to this service]",
   "description": "[human string]", ?
   "docsurl": "[URL reference for human documentation]", ?
+  "deprecated": { ?
+    "removaltime": "[RFC339 timestamp of when service will be removed]", ?
```

Comment:
Following the spirit of the main comment thread to follow the HTTP semantics shouldn't we call this `sunsettime` ??

Reply:
done

Reply:
per the call today we switched to "removaltime"